### PR TITLE
Fix ORG sampling bug

### PR DESCRIPTION
**Pull request by DrGlaucous**
Also included the full org3 wavetable.

Fixes a problem where the game would crash if an ORG3 song was loaded in-game without a large enough wavetable. Additionally, the drum initializer would load drum types based on the *last* ORG loaded, not the current one.

*(Example: if an org3 song is loaded after an org2, the game would look at the currently loaded org2's type when determining which drums to load)*

### CODE REVIEW

File: src/scene/game_scene.rs [hunk 1]
```diff
@@ -1245,16 +1245,7 @@ impl GameScene {
...
-                    && (npc.x - npc.hit_bounds.right as i32) < (bullet.x + bullet.enemy_hit_width as i32)
-                    && (npc.x + npc.hit_bounds.right as i32) > (bullet.x - bullet.enemy_hit_width as i32)
-                    && (npc.y - npc.hit_bounds.top as i32) < (bullet.y + bullet.enemy_hit_height as i32)
-                    && (npc.y + npc.hit_bounds.bottom as i32) > (bullet.y - bullet.enemy_hit_height as i32))
-                    || (npc.npc_flags.invulnerable()
-                        && (npc.x - npc.hit_bounds.right as i32) < (bullet.x + bullet.hit_bounds.right as i32)
-                        && (npc.x + npc.hit_bounds.right as i32) > (bullet.x - bullet.hit_bounds.left as i32)
-                        && (npc.y - npc.hit_bounds.top as i32) < (bullet.y + bullet.hit_bounds.bottom as i32)
-                        && (npc.y + npc.hit_bounds.bottom as i32) > (bullet.y - bullet.hit_bounds.top as i32));
+                let hit = npc.collides_with_bullet(bullet);
```
Comment (alula) [inline src/scene/game_scene.rs:1248]: could this be moved to a separate PR

Comment (alula) [inline src/scene/game_scene.rs:1248]: could this be moved to a separate PR?

**Review by alula — CHANGES_REQUESTED** [review]

File: src/scene/game_scene.rs (hunk 1 above)
Comment (DrGlaucous) [reply src/scene/game_scene.rs:1248]: Of course.
